invenio/b2share/modules/b2deposit/restful.py

```diff
@@ -216,13 +222,28 @@ def get(self, oauth, domain_name, **kwargs):
         record_list = []
         start = page_offset * page_size
         stop = start + page_size
+
         for record_id in record_ids[start : stop]:
             record_details = get_record_details(record_id, curr_user_email)
             record_list.append(record_details)
-        if stop < len(record_ids):
-            record_list.append('...') # continuation indicator
-        return record_list
 
+        if len(record_list) == 0:
```

Comment:
These are the cases
1- if it is empty => JSONIFY({})
2- if there is only one deposit => there are two cases:
.. 1- if user requests whit no page_offset -> JSONIFY(Deposit)
.. 2- if user added (unnecessary page_offset ) ->  JSONIFY({})
3- if there are more deposits:
.. 1- if user asked for first page_offset -> jsonify(Deposits)
.. 2- if user for other page_offset:
.... * if there is still more deposits in the requested page_offset: ->  jsonify(Deposits)
.... * if no more deposits in the requested page_offset, should return empty ->  jsonify({})